docs/topics/templates/document-attributes.adoc

```diff
@@ -31,7 +31,7 @@
 :Thorntail: WildFly Swarm
 :SpringBoot: Spring Boot
 :VertX: Eclipse Vert.x
-:Node: Node.js
+:NodeJS: Node.js
```

Comment:
This was changed recently on a lot of places. I don't think this change is correct.

Reply:
Also, I don't think that this change is appropriate within the scope of this PR.